public/docs/ts/latest/tutorial/toh-pt5.jade

```diff
@@ -479,16 +479,6 @@ code-example(format='').
   The colon (:) in the path indicates that `:id` is a placeholder to be filled with a specific hero `id`
   when navigating to the `HeroDetailComponent`.
 
-.l-sub-section
```

Comment:
Line 482 to 484 are relevant for Dart, please keep them and: just before this line, insert:
```
+ifDocsFor('dart')
```
and then indent lines 482 to 484.